apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -144,6 +146,8 @@ const querySchema = z.object({
 
 export type EventTypeSetupProps = RouterOutputs["viewer"]["eventTypes"]["get"];
 export type EventTypeSetup = RouterOutputs["viewer"]["eventTypes"]["get"]["eventType"];
+export type EventTypeAssignedUsers = RouterOutputs["viewer"]["eventTypes"]["get"]["eventType"]["children"];
+export type EventTypeHosts = RouterOutputs["viewer"]["eventTypes"]["get"]["eventType"]["hosts"];
```

Comment:
"No need to declare it again; the type `Host` is already present in file. Please export it."

Reply:
Yes , i tried using type - Host already defined in event-types-single-view.tsx.
Since the prop-priority is non-null, it is breaking other components.